base_conditional_image/models/abstract_image.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from openerp import fields, models
+from openerp.tools.safe_eval import safe_eval
+
+
+class AbstractImageStorage(models.AbstractModel):
+    _name = 'abstract.conditional.image'
```

Comment:
Is a there a reason why there is no matching between _name and class name ?
Not blocking